core/trino-spi/src/main/java/io/trino/spi/connector/RowChangeParadigm.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.connector;
+
+public enum RowChangeParadigm
+{
+    /**
+     * The JDBC paradigm - - requires just the rowId and the changed columns
```

Comment:
Remove the JDBC and Hive/Iceberg part since that makes it seem like these are specific to those implementations. Instead, we can explain more about how it works.

Reply:
Removed/changed.